app/controllers/likes_controller.rb

```diff
@@ -1,2 +1,11 @@
 class LikesController < ApplicationController
```

Comment:
- [ ] It would be better if we are able to add users to our database, however, I'm unable to add user because we have validated our post counter in our model and the post_counter is not included in our code in creating a user `first_user = User.create(name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Mexico.')`
![image](https://user-images.githubusercontent.com/46388113/186636586-7eca623f-deb8-45f8-b27c-efff07e558d9.png)

Reply:
@addod19 Could you please see my message on Slack? I have a doubt here. Thanks!

Reply:
@addod19 The solution that I have found is to disable the validate for the field post_counter. With this, you are able to add a user, just like the screenshot below. Please confirm if this is your requirement above. Thank you!

![image](https://user-images.githubusercontent.com/93770917/186737851-26b2b61a-65c0-4d3e-8358-671c048434cb.png)

![image](https://user-images.githubusercontent.com/93770917/186738411-b3e0e784-1efa-40ff-b8af-0c23ece325e5.png)

Reply:
Awesome!!! Looks good to me since we are able to save data into the database :man_technologist:

Reply:
Ok. I will submit the Project again for a new review. Thanks!

Reply:
Well done